.github/workflows/pytest.yaml

```diff
@@ -108,13 +90,13 @@ jobs:
         version: ${{ env.GAMS_VERSION }}
         license: ${{ secrets.GAMS_LICENSE }}
 
-    - name: Install ixmp and dependencies
-      working-directory: ixmp
-      run: pip install .
-
     - name: Install Python package and dependencies
-      working-directory: message_ix
-      run: pip install .[tests]
+      # By default, the below installs ixmp from the main branch. To run against
+      # other code, e.g. other branches for open PRs), temporarily edit as
+      # appropriate. DO NOT merge such changes to `main`.
+      run: |
+        pip install --upgrade "ixmp @ git+https://github.com/iiasa/ixmp.git@main"
```

Comment:
Out of curiosity: 'edit as appropriate' means to replace 'main' in the command with the name of the branch that should be tested?

Reply:
Yes, exactly.